### ISSUE TITLE
Feature request: Blur assistant when previewing images

### ISSUE THREAD

**Issue by DaCor1** [mapillary/mapillary_issues#630]
It would be a good feature, if, when I am reviewing images, I could tap the locations on images where blurring is definitely needed. I don't know how this might work, but in essence, these tapped locations on images, would point your algorithm in the right direction when choosing where to blur

**Comment by jesolem** [comment]
We could even blur the original on device. Would be a neat solution.

**Comment by DaCor1** [comment]
True, tap to locate blur spot, pinch to resize the blur, double tap to
change shape of blur...... or something like that ;)

**Comment by iav** [comment]
And don't forget an easy way to fast change image zoom and blur size and position. Beware fingers are not very accurate tool on a touch-screens, allow large movevents.

**Comment by jesolem** [comment]
Let's see how we can plan something like this in the apps roadmap. -> @peterneubauer

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
